### ISSUE TITLE
Clarify MIDI Start and Continue behaviour

### ISSUE THREAD

**Issue by ggVGc** [datomusic/duo-imxrt#162]
**Current behaviour**

**Comment by DavidMenting** [comment]
We've chosen to implement `start` as `continue` when pressing the play button as there is no clear first step on the DUO. This in contrast to the Volca's that have a linear sequencer. You can use the BOOST button to choose which step to continue from.

The behaviour when receiving a MIDI `start` or `continue` message is also deliberate but the case can be made to have `start` restart from the first step. Again, as the DUO has no clear first step I don't think there is a clear benefit.

**Comment by ggVGc** [comment]
@DavidMenting Yeah, I think that makes sense.
The only remaining question for me is why we output `Start` both when receiving `Start` and `Continue`. I think it would make sense to either output `Continue` (as we do when starting from play button), or repeat the incoming message.

**Comment by DavidMenting** [comment]
you are right. Passing on the received message seems the most logical thing to do
